WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -256,6 +256,46 @@ public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int? fe
 		return smartTx.Transaction.ToHex();
 	}
 
+	/// <summary>
+	/// Unsafe, because no matter how big fee the user chooses, Wasabi will build the transaction.
+	/// Potentially, the user can burn his money using this method, so be careful!
+	/// </summary>
+	[JsonRpcMethod("buildunsafetransaction")]
+	public string BuildUnsafeTransaction(PaymentInfo[] payments, OutPoint[] coins, int? feeTarget = null, decimal? feeRate = null, string? password = null)
+	{
+		Guard.NotNull(nameof(payments), payments);
```

Comment:
This seems to me a code duplication, is it?

Reply:
Yes, it is and it is by design. Check this comment https://github.com/zkSNACKs/WalletWasabi/pull/12124#issuecomment-1866782796.

First, I implemented this by adding one more boolean parameter to the Build RPC method, but the parameters are getting out of hand, so it was recommended not to introduce more parameters.

Reply:
I agree with that. What I am saying is that if you can extract the code duplication into a private method and use that.

Reply:
OK, done in [1cf1372](https://github.com/zkSNACKs/WalletWasabi/pull/12124/commits/1cf13723a542e86309893cd5844e7495189b566b)

But I don't think refactoring is the responsibility of this PR. Now there are changes in code where previously were not. This makes the PR harder to test and harder to get it merged (you have to test both `build` methods now to make sure I didn't introduce a bug or a change in their behavior). 
Also refactoring got nothing to do with fixing the issue, so IMO this could have waited for another PR.

_Anyways, my two sats._

Reply:
Yes, I agree, refactoring should be always in separate PR. But here you are adding new code that copy pastes and that is a duplication we must avoid. Why?
- It is error-prone if one changes the logic, it has to be changed at multiple places - usually, it is hard to see and forget. 
- Less code - improve readability.
- Extracting method - improve modularity.